### PR TITLE
SWAT vest !!!DOES NOT BUFF SWAT!!!

### PR DESCRIPTION
**Pull request by Bawhoppen**
All it does is add a tacticool vest version of the Mk.1 Swat armor that is dropped by some simple mobs and 2 pairs are found on the meta whiteship. 
I will remind you that SWAT armor is only ever so slightly better than normal armor anyways.

It does not make them orderable by cargo.
It does not buff any armor in an away.
Please give me a chance.

### CODE REVIEW

File: code/modules/awaymissions/corpse.dm [hunk 1]
```diff
@@ -406,7 +406,7 @@
 /obj/effect/mob_spawn/human/nanotrasensoldier
```
Comment (francinum) [inline code/modules/awaymissions/corpse.dm]: These are spawned by the Orion Trail arcade emag setting, being a minor buff to emags.

Comment (Bawhoppen) [inline code/modules/awaymissions/corpse.dm]: That's weird but I think it's fine.

Comment (francinum) [inline code/modules/awaymissions/corpse.dm]: I'm being pedantic and pointing fun, it's perfectly fine.

**Comment by AnturK** [comment]
Pictures please.

**Comment by Bawhoppen** [comment]
ss 2017-03-18 at 12 20 24

**Comment by KorPhaeron** [comment]
@optimumtact

File: code/modules/clothing/suits/armor.dm [hunk 2]
```diff
@@ -139,6 +139,17 @@
...
 
+/obj/item/clothing/suit/armor/swat
+	name = "MK.I SWAT Vest"
+	armor = list(melee = 40, bullet = 30, laser = 30,energy = 30, bomb = 50, bio = 90, rad = 20, fire = 100, acid = 100)
+	desc = "A lightweight vest variant of the infamous tactical suit."
+	icon_state = "swatvest"
+	item_state = "armor"
+	allowed = list(/obj/item/weapon/gun,/obj/item/ammo_box,/obj/item/ammo_casing,/obj/item/weapon/melee/baton,/obj/item/weapon/restraints/handcuffs,/obj/item/weapon/tank/internals,/obj/item/weapon/kitchen/knife/combat)
+	strip_delay = 120
+	resistance_flags = FIRE_PROOF | ACID_PROOF
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/clothing/suits/armor.dm]: Excuse you

File: code/modules/clothing/suits/armor.dm [hunk 3]
```diff
@@ -139,6 +139,17 @@
...
 		return 1
 
+/obj/item/clothing/suit/armor/swat
+	name = "MK.I SWAT Vest"
+	armor = list(melee = 40, bullet = 30, laser = 30,energy = 30, bomb = 50, bio = 90, rad = 20, fire = 100, acid = 100)
+	desc = "A lightweight vest variant of the infamous tactical suit."
+	icon_state = "swatvest"
+	item_state = "armor"
+	allowed = list(/obj/item/weapon/gun,/obj/item/ammo_box,/obj/item/ammo_casing,/obj/item/weapon/melee/baton,/obj/item/weapon/restraints/handcuffs,/obj/item/weapon/tank/internals,/obj/item/weapon/kitchen/knife/combat)
+	strip_delay = 120
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/clothing/suits/armor.dm]: Excuse You

File: code/modules/clothing/suits/armor.dm [hunk 4]
```diff
@@ -139,6 +139,17 @@
 	if (prob(hit_reflect_chance))
 		return 1
 
+/obj/item/clothing/suit/armor/swat
+	name = "MK.I SWAT Vest"
+	armor = list(melee = 40, bullet = 30, laser = 30,energy = 30, bomb = 50, bio = 90, rad = 20, fire = 100, acid = 100)
+	desc = "A lightweight vest variant of the infamous tactical suit."
+	icon_state = "swatvest"
+	item_state = "armor"
+	allowed = list(/obj/item/weapon/gun,/obj/item/ammo_box,/obj/item/ammo_casing,/obj/item/weapon/melee/baton,/obj/item/weapon/restraints/handcuffs,/obj/item/weapon/tank/internals,/obj/item/weapon/kitchen/knife/combat)
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/clothing/suits/armor.dm]: Inherit from suit/armor

**Review by ExcessiveUseOfCobblestone — CHANGES_REQUESTED** [review]

**Comment by ExcessiveUseOfCobblestone** [comment]
Also doesn't meet requirements of Contributing, needs to have a "purpose" section.

**Comment by PKPenguin321** [comment]
ok cool but i dont get why youre so obsessed with these things

File: code/modules/clothing/suits/armor.dm (hunk 3 above)
Comment (Bawhoppen) [inline code/modules/clothing/suits/armor.dm]: Thanks for the helpful comment

File: code/modules/clothing/suits/armor.dm (hunk 4 above)
Comment (Bawhoppen) [inline code/modules/clothing/suits/armor.dm]: It's a different set of allowed items (though arguably it doesn't have to be)

**Comment by Bawhoppen** [comment]
@PKPenguin321 the simple mobs and stuff have SWAT helms but just normal armor vests and it has bugged me to no end also I had this cool sprite ready

**Comment by optimumtact** [comment]
Don't ping me in this shit

**Comment by Cheridan** [comment]
powercreep

File: code/modules/clothing/suits/armor.dm (hunk 4 above)
Comment (ExcessiveUseOfCobblestone) [inline code/modules/clothing/suits/armor.dm]: Oh it also holds guns my b

File: code/modules/clothing/suits/armor.dm (hunk 3 above)
Comment (ExcessiveUseOfCobblestone) [inline code/modules/clothing/suits/armor.dm]: 2 minutes to remove a vest when objectively better armor is at 80 secs seems a bit silly.

**Comment by ExcessiveUseOfCobblestone** [comment]
I'm not sure if anyone has asked you this, but if you want them to be aesthetically consistent why not just have a new type of `armor/vest` like "alt2" that is just a reskin instead of making the armor objectively better in what i'm assuming to be every way [but not a buff, honest!]?

File: code/modules/clothing/suits/armor.dm (hunk 3 above)
Comment (Xhuis) [inline code/modules/clothing/suits/armor.dm]: This is 12 seconds, isn't it? BYOND uses deciseconds.

File: code/modules/clothing/suits/armor.dm (hunk 2 above)
Comment (Xhuis) [inline code/modules/clothing/suits/armor.dm]: This just means that they can't be destroyed by fire and acid, not protects the wearer.

**Comment by Bawhoppen** [comment]
@ExcessiveUseOfCobblestone when I say buff I am referring to how in my old PRs I used to always try and buff the swat armor whilst adding the new vest, this time I did not

**Comment by Bawhoppen** [comment]
Please make my day and merge this

File: code/modules/clothing/suits/armor.dm (hunk 3 above)
Comment (Supermichael777) [inline code/modules/clothing/suits/armor.dm]: yes

File: code/modules/clothing/suits/armor.dm (hunk 2 above)
Comment (optimumtact) [inline code/modules/clothing/suits/armor.dm]: There's no reason for this though

**Comment by Shadowlight213** [comment]
Armor comparison with some other armors that I found interesting here.
Swat
melee = 40, bullet = 30, laser = 30,energy = 30, bomb = 50, bio = 90, rad = 20, fire = 100, acid = 100
resistance_flags = FIRE_PROOF | ACID_PROOF
Hos
melee = 30, bullet = 30, laser = 30, energy = 10, bomb = 25, bio = 0, rad = 0, fire = 70, acid = 90
Captain's carapace
melee = 50, bullet = 40, laser = 50, energy = 10, bomb = 25, bio = 0, rad = 0, fire = 100, acid = 90
resistance_flags = FIRE_PROOF
Rd's hardsuit
melee = 30, bullet = 5, laser = 10, energy = 5, bomb = 100, bio = 100, rad = 60, fire = 60, acid = 80
resistance_flags = ACID_PROOF | FIRE_PROOF

**Comment by Bawhoppen** [comment]
@shadowlight213 Standard armor is equivalent to HOS's I believe

File: code/modules/clothing/suits/armor.dm (hunk 2 above)
Comment (Bawhoppen) [inline code/modules/clothing/suits/armor.dm]: Other swat gear has it but if you want it removed that's fine, I am away from a computer for awhile

**Comment by Bawhoppen** [comment]
Wait ignore my previous comment I skimmed

**Comment by lzimann** [comment]
@tgstation/commit-access Thoughts on this? Personally, I think swat gear is eh...

**Comment by Bawhoppen** [comment]
Please don't crush my hopes and dreams

**Comment by jughu** [comment]
Its merge or be merged

**Comment by KorPhaeron** [comment]
Realistically I don't think anyone is ever going to merge this but @tgstation/commit-access is free to reopen and prove me wrong at any time

**Comment by Bawhoppen** [comment]
I will be back to my comp in like 4 days, I urge you to reopen so I can make my case to a maintainer in IRC then

**Comment by Bawhoppen** [comment]
@oranges If I had a computer and could edit a video, I'd make a reply video with the scene from  Rick and Morty, where Roy is dying from cancer and says "I'm not ready to die..." and then his wife says "You're not going to!" and I'd edit the SWAT vest sprite over Roy's face and my avatar over his wife's.

**Comment by Razharas** [comment]
Ok bawhoppen bawwed real hard so lets try it out

**Comment by optimumtact** [comment]
Still :-1:

**Comment by Razharas** [comment]
I havent merged stuff in a while and this looks harmless enough

**Comment by Bawhoppen** [comment]
# As I promised I will never make an armor related change again

**Pull request merged**